tests/unit/FilingHistoryList2.spec.ts

```diff
@@ -106,6 +107,30 @@ filings.forEach((filing: any, index: number) => {
       expect(item.correctionFilingId).toBe(filing.correctionFilingId)
     })
 
+    itIf(isIncorporationApplication(filing))('TP Hide Documents button for IAs', async () => {
+      expect(vm.getFilings.length).toBe(1) // sanity check
+      const expandBtn = wrapper.find('.expand-btn')
+      rootStore.$state = {
+        ...rootStore.$state,
+        entityStatus: EntityStatus.FILED_CONTINUATION_IN // Set isBootstrapFiling to true for the purpose of the test
```

Comment:
Shouldn't this be `FILED_INCORP_APP` since you're testing for an Incorporation Application?

Reply:
It could be any of the 4 bootstrap filings.

Reply:
I was thinking since on line 110, it says for IAs.

Reply:
Thank you. I will move this test to FilingHistoryList1.spec.ts. Since all testing files for FilingHistoryList2.spec.ts are with business identifier, instead of temp TP.

Reply:
Thanks, Resolving.